src/components/Book.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import Button from './Button';
+
+const Book = (props) => {
+  const name = props;
+  return (
+    <div key={name.data[0].id}>
+      <h1>{name.data[0].type}</h1>
+      <h2>{name.data[0].title}</h2>
+      <h2>{name.data[0].author}</h2>
+      <Button />
+      <h1>{name.data[1].type}</h1>
+      <h2>{name.data[1].title}</h2>
+      <h2>{name.data[1].author}</h2>
+      <Button />
+      <h1>{name.data[2].type}</h1>
+      <h2>{name.data[2].title}</h2>
+      <h2>{name.data[2].author}</h2>
+      <Button />
+
+    </div>
+  );
+};
```

Comment:
- [ ] In the previous review, it was pointed out that you use prop validation for this particular component. Since this is not addressed, we'll need to look into it one more time. To get an idea of what `prop validation` or `type checking` is, we can take a look at this [guide](https://reactjs.org/docs/typechecking-with-proptypes.html).